src/query/parser/promql/parse.go

```diff
@@ -160,11 +160,15 @@ func (p *parseState) walk(node pql.Node) error {
 			}
 		}
 
-		op, err := NewFunctionExpr(n.Func.Name, argValues, stringValues)
+		op, ok, err := NewFunctionExpr(n.Func.Name, argValues, stringValues)
```

Comment:
Would prefer this check here for n.func.name and return nil if it's a sort, rather than adding awkward third return param